scripts/gpu_support/nvidia/install_cuda_and_libraries.sh

```diff
@@ -239,8 +239,8 @@ for EASYSTACK_FILE in ${TOPDIR}/easystacks/eessi-*CUDA*.yml; do
       echo_green "all installations at ${EESSI_SITE_SOFTWARE_PATH}/software/... succeeded!"
     fi
 
-    # clean up tmpdir
-    rm -rf "${tmpdir}"
+    # clean up tmpdir content
+    rm -rf "${tmpdir}"/*
```

Comment:
Shouldn't we then remove `${tmpdir}` after the loop?

Reply:
https://github.com/EESSI/software-layer/pull/814#discussion_r1846338150

Reply:
Yes, but that approach only removes the contents of `$tmpdir`, never `$tmpdir` itself. This can be done outside the loop, see #816